src/cli/package.rs

```diff
@@ -95,7 +95,8 @@ impl Packager {
 
             let json_file = fs::read_to_string(json_file_path)?;
 
-            serde_json::from_str(&json_file)?
+            // if the json file does not contain an Object at the top level, we can't parse it
```

Comment:
What does unwrap_or_default do here?

Reply:
@Connoropolous  If `Err`, uses the Default value for the type getting unwrapped to. So in this case the default is just `Object::new()`, or the empty Map.

Reply:
Cool